ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/MiniProtocol/BlockFetch/ClientInterface.hs

```diff
@@ -333,10 +348,10 @@ mkBlockFetchConsensusInterface
       | otherwise
       = preferAnchoredCandidate bcfg ours cand
 
-    compareCandidateChains :: AnchoredFragment (Header blk)
-                           -> AnchoredFragment (Header blk)
+    compareCandidateChains :: AnchoredFragment (HeaderWithTime blk)
+                           -> AnchoredFragment (HeaderWithTime blk)
                            -> Ordering
     compareCandidateChains = compareAnchoredFragments bcfg
 
-    headerForgeUTCTime = slotForgeTime . headerRealPoint . unFromConsensus
+    headerForgeUTCTime = slotForgeTime . headerRealPoint . hwtHeader . unFromConsensus
```

Comment:
Add a comment here that this is the place to adapt once https://github.com/IntersectMBO/ouroboros-network/pull/5009 is integrated into Consensus?

Reply:
The `blockForgeUTCTime` definition you mean?

Reply:
TODO: change the implementation of `headerForgeUTCTime` to use the `HeaderWithTime` annotation, and add the comment Alex mentioned above `blockForgeUTCTime`.